base.rb

```diff
@@ -0,0 +1,48 @@
+require 'bundler'
+require 'yaml'
+require 'sinatra/base'
+require "sinatra/multi_route"
+require 'securerandom'
+require 'active_support/core_ext/hash'
+require 'active_support/inflector'
+require './lib/tendrl'
```

Comment:
It shouldn't be necessary to use relative paths to the lib directory for `require`. It must be possible to add the `lib` directory to the ruby library path environment variable at startup, via config.ru or equivalent. I've previously done this in the unicorn startup file. If puma and shotgun are capable of evaluating config.ru, then it should be added there.